target/linux/ath79/dts/qca9557_zyxel_nbg6616.dts

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	compatible = "zyxel,nbg6616", "qca,qca9557";
+	model = "ZyXEL NBG6616";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power: power {
+			label = "nbg6616:green:power";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1 {
+			label = "nbg6616:green:usb1";
+			gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port0>;
+		};
+
+		usb2 {
+			label = "nbg6616:green:usb2";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port1>;
+		};
+
+		wlan2g {
+			label = "nbg6616:green:wifi2g";
```

Comment:
Please either call it wifi2g or wlan2g, but do it consistently for label and node name. Same below.

Reply:
Hmm, to keep the label in sync with ar71xx, you should change the node name.

Reply:
done